src/Kiota.Builder/Writers/Python/PythonRelativeImportManager.cs

```diff
@@ -23,19 +23,19 @@ public override (string, string, string) GetRelativeImportPathForUsing(CodeUsing
             return (string.Empty, string.Empty, string.Empty);//it's an external import, add nothing
         var (importSymbol, typeDef) = codeUsing.Declaration?.TypeDefinition is CodeElement td ? td switch
         {
-            CodeFunction f => (f.Name.ToFirstCharacterLowerCase(), td),
-            _ => (td.Name.ToFirstCharacterUpperCase(), td),
+            CodeFunction f => (f.Name, td),
+            _ => (td.Name, td),
         } : (codeUsing.Name, null);
 
         if (typeDef == null)
             return (importSymbol, codeUsing.Alias, "."); // it's relative to the folder, with no declaration (default failsafe)
         var importPath = GetImportRelativePathFromNamespaces(currentNamespace,
             typeDef.GetImmediateParentOfType<CodeNamespace>());
         if (string.IsNullOrEmpty(importPath))
-            importPath += codeUsing.Name;
+            importPath += codeUsing.Name.ToSnakeCase(); // TODO: review the logic happening for those imports
```

Comment:
Here is where things are a bit hairy.
I have had to leave those renamings in the Writers, as they refer to the file name which is information not known from the `CodeDOM`, please review carefully those changes and let me know if a different implementation is preferred.

Reply:
Yeah this is a bit of a dead angle in kiota's DOM design. We only considered the logical layout of the code, not the implications for physical files. This works well in languages where file naming conventions map exactly to symbols naming conventions (like C#), but it shows limitations in other cases. We later on introduced a notion of code files (which are effectively bundles of multiple symbols that need to be written out to the same file). I'm afraid generalizing code files to every symbol (wrapping them up) or introducing a concept for that aspect is going to lead to a LOT of refactoring. I'm ok with leaving things as is for now until we have a better solution. Thoughts?

Reply:
> I'm ok with leaving things as is for now until we have a better solution.

As long as this doesn't start to break often I'm ok with this assessment 👍